test/jasmine/tests/hover_label_test.js

```diff
@@ -4177,11 +4194,12 @@ describe('hover distance', function() {
         it('responds to hoverdistance change from default to 0 (part 2)', function() {
             var gd = document.getElementById('graph');
             var evt = { xpx: 475, ypx: 155 };
-            Plotly.relayout(gd, 'hoverdistance', 0);
-
-            Fx.hover('graph', evt, 'xy');
+            Plotly.relayout(gd, 'hoverdistance', 0)
+            .then(function() {
+                Fx.hover('graph', evt, 'xy');
 
-            expect(gd._hoverdata).toEqual(undefined);
+                expect(gd._hoverdata).toEqual(undefined);
+            });
```

Comment:
`done`/`failTest`

Reply:
Done in https://github.com/plotly/plotly.js/pull/5340/commits/1694bb407d4138591eda35d31f4b4e176af17282.